From d7451484e7b947b79c30dff4304a599f6c1e470e Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 28 Aug 2020 17:54:22 -0700
Subject: [PATCH 30/47] Cite Rishi et al in tz-link

* tz-link.html (Costs and benefits of time shifts):
Add citation.  (Thanks to Arthur David Olson.)
---
 tz-link.html | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/tz-link.html b/tz-link.html
index 8141d71..6dcb654 100644
--- a/tz-link.html
+++ b/tz-link.html
@@ -905,7 +905,8 @@ href="http://sohma.armada.mil.uy/index.php/servicios/datos-astronomicos" hreflan
 <h2 id="costs">Costs and benefits of time shifts</h2>
 <p>Various sources argue for and against daylight saving time and time
 zone shifts, and many scientific studies have been conducted. This
-section summarizes reviews of scientific literature in the area.</p>
+section summarizes reviews and position statements based on
+scientific literature in the area.</p>
 <ul>
 <li>Carey RN, Sarma KM.
 <a href="https://bmjopen.bmj.com/content/7/6/e014319.long">Impact of
@@ -923,6 +924,13 @@ doi:<a href="https://doi.org/10.5547/01956574.39.2.thav">10.5547/01956574.39.2.t
 This analyzes research literature and concludes, "Electricity savings
 are larger for countries farther away from the equator, while
 subtropical regions consume more electricity because of DST."</li>
+<li>Rishi MA, Ahmed O, Barrantes Perez JH <em>et al</em>.
+<a href="https://jcsm.aasm.org/doi/10.5664/jcsm.8780">Daylight saving time:
+an American Academy of Sleep Medicine position statement</a>.
+<em>J Clin Sleep Med.</em>
+2020;<a href="https://doi.org/10.5664/jcsm.8780">10.5664/jcsm.8780</a>.
+This argues for permanent standard time due to health risks of both
+DST transitions and permanent DST.</li>
 <li>Roenneberg T, Winnebeck EC, Klerman EB.
 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6692659/">Daylight
 saving time and artificial time zones &ndash; a battle between
-- 
1.8.3.1

